javascript/GoogleMapField.js

```diff
@@ -87,13 +101,16 @@
 		}
 
 		// Populate the fields to the current centre
-		updateField(map.getCenter(), true);
+		google.maps.event.addListenerOnce(map, 'idle', function(){
+			updateField(map.getCenter(), JSON.stringify(map.getBounds().toJSON()), true);
```

Comment:
This line originally to update the fields when first time add a new record. Since it add bounds, the bounds will not be available until the map tiles loaded which will be triggered by 'idle' event. Not this will only be triggered once after map is loaded.